index.js

```diff
@@ -7,8 +7,8 @@ var processPath = require.resolve('process/browser.js');
 var isbufferPath = require.resolve('is-buffer')
 var combineSourceMap = require('combine-source-map');
 
-function getRelativePath(fullPath, fromPath) {
-  var relpath = path.relative(path.dirname(fromPath), fullPath);
+function getRelativeRequirePath(fullPath, fromPath) {
+  var relpath = path.posix.relative(path.dirname(fromPath), fullPath);
```

Comment:
`path.posix` was only introduced in node-v0.11.15, so this needs a fallback or to use a shimmed version of `path` like path-posix or similar.